src/main/resources/mocks/StatusDescription.json

```diff
@@ -0,0 +1,249 @@
+{
+  "results": [
+    {
+      "statusId": 1,
```

Comment:
for consistency of the data can we have id instead of statusId? not biggy but I think if we have the id of the table to be in this way it will be more consistent. What do you think?

Reply:
Done.